docker/cellranger/README.md

```diff
@@ -0,0 +1,20 @@
+# Steps for making Docker image:
+
+We are using cell ranger from https://github.com/10XGenomics/cellranger . 
+This is currently Cellranger 2.1.1:w
+
+This is licensed with an MIT license and so is most compatible with HCA data.
+
+## Build docker image
+
+Build container:
+
+`docker build .`
+
+## Interactively work with image
+
+`docker run -it <image> /bin/bash`
+
+## Verify with the following command:
+
+`docker run <image>`
```

Comment:
I'd like some kind of basic test that shows cellranger working. Ideally that would be on testing data, but if you're stretched I'd take an invocation of the binary that prints help or something. :-)

Reply:
How about this existing test @ambrosejcarr? https://github.com/HumanCellAtlas/skylab/blob/master/test/trigger_cellranger_test.sh

Reply:
We should provide an example inputs json file next to the cellranger wdl file as part of this PR. We should probably also set up a some test automation but that could happen in a later PR.

Reply:
For local testing, I was using a `transcriptome_tar_gz` file in one of our team gcloud buckets. But I believe it was originally downloaded from cellranger and required agreeing to their software license, so we would need to provide our own reference in the example inputs if my understanding is correct here.

Reply:
Hmm, okay, @ambrosejcarr or @mckinsel, do you know if we have our own transcriptome tar file that we could use or is this something that has to come from 10x?

Reply:
You can make your own using `cellranger mkref`, which I'm pretty sure Ambrose has done a few times. I thought they had a small one available on their website, but you now have to agree to their privacy policy, which I do not.

Reply:
Ok! For now I put the cellranger reference in one of our private mint team gcloud buckets, and we can replace that in the example inputs when we have a reference of our own, which I think can be a separate ticket.